string-manipulation-2/src/capitalize-word.js

```diff
@@ -0,0 +1,12 @@
+/* eslint-disable no-unused-vars */
+function capitalizeWord(word) {
+  var endOfWord = word.substring(1, word.length);
+  endOfWord = endOfWord.toLowerCase();
+  var firstLetter = word[0].toUpperCase();
+  var newWord = firstLetter + endOfWord;
+
+  if (word.toLowerCase() === 'javascript') {
+    return 'JavaScript';
+  }
```

Comment:
It's best to get this special case out of the way right at the beginning of the function.